composer.json

```diff
@@ -66,6 +66,15 @@
             },
             "drupal/amptheme": {
                 "Don't use network for loading logo information": "https://www.drupal.org/files/issues/amptheme-do_not_use_the_network_for_loading_logo_information-2753089-7.patch"
+            },
+            "drupal/entity_browser": {
+                "Provide inline entity form FieldWidgetDisplay": "https://www.drupal.org/files/issues/2858438_3.patch"
```

Comment:
This patch contains a var_dump, please remove this. The maintainer also needs other changes, fix those issues as well, to get that merged. See: https://www.drupal.org/node/2858438